### ISSUE TITLE
Annotation notifications within discourse

### ISSUE THREAD

**Issue by kgish** [kgish/discourse-annotator-plugin#29]

**Comment by kgish** [comment]
Please explain in more detail what this means.

**Comment by Socioneers** [comment]
Wim and decided this is not a prio, for **future** it would be great to get notifications when someone comments on your annotation.

**Comment by Socioneers** [comment]
What could be possible to indicate new amendementen in the the filter menu at the front page?

**Comment by kgish** [comment]
Not sure I understand, please explain.

**Comment by Socioneers** [comment]
Right now amendment actions do not turn up in discourse notification or filtering options. Is there a way to get amendeer in put or topics in de filterlist/ topmenu's. See examples:
<img width="569" alt="screen shot 2016-10-10 at 13 22 06 2" src="https://cloud.githubusercontent.com/assets/11832800/19234620/f7ddd50a-8eec-11e6-8030-838490df552b.png">
<img width="742" alt="screen shot 2016-10-10 at 13 23 28" src="https://cloud.githubusercontent.com/assets/11832800/19234621/f7e22470-8eec-11e6-9cc8-b5bcc09ac83f.png">

**Comment by kgish** [comment]
Yes, the same way I introduced the [Mijn berichten]-button can be used.

**Comment by kgish** [comment]
This is a difficult problem, and I will need around 4 hours to research it properly. After that the additional effort depends on the complexity of the solution.

Shall I proceed?

**Comment by Socioneers** [comment]
No lets wait
